data/resources/ui/content-message-row.ui

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Oops, that somehow crept in. I guess while rebasing? I really don't know.

Reply:
I was getting this problem all the time and I couldn't figure out what it was, then recently I realized that it was Gnome Builder's auto saving option (enabled by default). So, if you use Builder that could be the reason. Otherwise it was there from the rebase.